src/resources/rides.jsx

```diff
@@ -0,0 +1,76 @@
+import { 
+    List, 
+    ReferenceField,
+    TextField, 
+    Datagrid,
+    Edit,
+    SimpleForm,
+    DateField,
+    ReferenceInput,
+    TextInput,
+    DateTimeInput,
+    NumberInput,
+    Create,
+    SelectField,
+    SelectInput
+} from "react-admin";
+
+const rideFilters = [
+  <TextInput source="q" label="Search" alwaysOn />,
+];
+
+const states = [
```

Comment:
the id is off by one, it should start with 0. also in all other enums I would assume.

Reply:
i know already fixed